code/datums/martial/jujitsu.dm

```diff
@@ -0,0 +1,114 @@
+#define TAKEDOWN_COMBO "DD"
+#define JUDO_THROW "DHHG"
+
+/obj/item/book/granter/martial/jujitsu
```

Comment:
Yeah, I don't think giving sec this shit is good, it just brings back stuncombat without the weaknesses a stunbaton had. NSV is a different server with different needs

Reply:
I agree with zesko for once

Reply:
Any criticisms apart from it affecting golems/space suit wearers? This could be fixed by having it only affect people who specifically have stamina damage already.

It's worse than just using a stun baton (2 clicks baton vs 3 for a slowdown + Takedown combo) so I don't see at all how that meshes with the "brings back stun combat" remark.